addons/test_mail/tests/test_message_management.py

```diff
@@ -36,6 +36,7 @@ def test_mail_resend_workflow(self):
                 [{'partner': partner, 'type': 'email', 'status': 'exception'} for partner in self.partners],
                 message_info={'message_type': 'notification'},
                 sim_error='connect_failure'):
+            self.connect_mocked.side_effect = Exception("Some exception")
```

Comment:
shouldn't you do:
```suggestion
            def _connect(self):
                raise Exception("Some exception")
            self.connect_mocked.side_effect = _connect
```

I think it will indeed fail, but for the wrong reason